### ISSUE TITLE
The NewVertex dialog (graph) is not themed, accessible and localizable

### ISSUE THREAD

**Issue by jinglouMSFT** [microsoft/AzureStorageExplorer#1280]
The new vertex dialog for Cosmos DB graph needs to be made accessible and localizable.  The Remove image and the Add Property image can hardly be seen in the dark theme.  The text with the Add Property image is currently not localizable.  It's also not accessible as a table.    

 
![image](https://user-images.githubusercontent.com/10081483/55844873-4f45b800-5af4-11e9-9e7a-bf547a7ab217.png)

**Comment by aliuy** [comment]
Azure Storage Explorer support for Azure Cosmos DB has been discontinued as of v1.24.0. We recommend users to use Azure Cosmos DB’s Data Explorer, which can be found in [Azure Portal](https://portal.azure.com/) or as a stand-alone version in http://cosmos.azure.com/ for the best Azure Cosmos DB experience. You can also find additional Azure Cosmos DB tooling here: [Azure Cosmos DB Tools](https://developer.azurecosmosdb.com/tools)
